yarn-project/noir-contracts/src/contracts/easy_zk_token_contract/src/main.nr

```diff
@@ -36,9 +36,9 @@ contract EasyZkToken {
         let storage = Storage::init();
         let balances = storage.balances;
 
-        context = balances.at(owner).add(context, initial_supply, owner);
+        balances.at(owner).add(&mut context, initial_supply, owner);
```

Comment:
Yay, new syntax! Just to check: the compiler will complain if we don't include this `&mut` keyword, right?